test.ps1

```diff
@@ -11,6 +11,7 @@ param (
 Set-StrictMode -Version Latest
 $ErrorActionPreference = 'Stop'
 # assigning error view explicitly to change it from the default on PowerShell 7
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseDeclaredVarsMoreThanAssignments", Scope="Function", Target="*")]
```

Comment:
I would keep this Supression on specific variables with targetting, otherwise keeping the test running on the repo will give a false sense of security as it actually will not report on a lot of possible issued.